### ISSUE TITLE
CSV reporter doesn't show baseline comparison

### ISSUE THREAD

**Issue by hasufell** [Bodigrim/tasty-bench#36]
https://github.com/Bodigrim/tasty-bench/blob/master/src/Test/Tasty/Bench.hs#L1574

I'd argue what you want here is something like: https://gist.github.com/hasufell/3710e86fcdfcd5f2c641e7d0a57e30ce

So: results for both and a column showing the percentage diff just like the console roporter

**Comment by Bodigrim** [comment]
I think this kind of CSV processing is better outsourced to external tools, see AWK incantations at https://github.com/Bodigrim/tasty-bench#comparison-against-baseline

**Comment by hasufell** [comment]
You're already doing CSV processing when calculating the baseline percentage in the console reporter, no?

That can be done with awk too.

**Comment by Bodigrim** [comment]
The console reporter is interactive, so there is a tangible benefit from displaying difference straight away, without third-party tools. CSV reporter is not interactive, so there is less added benefit from combining reports inside of `tasty-bench` instead of doing it outside. 

I'm not necessarily opposed to the idea, it's just a low priority from my perspective and I cannot quite calculate all its consequences.

**Comment by hasufell** [comment]
> instead of doing it outside

Well, I'd rather avoid using awk. I'm not opposed to doing it outside either. That may allow to do more processing and include heavier dependencies like [tapioca](https://hackage.haskell.org/package/tapioca). But it may as well be a quick patch... I don't know.

**Comment by Bodigrim** [comment]
I'd rather make `tasty-bench` leaner, not larger. Given that nobody is blocked by this feature and nobody is particularly interested to work on it, closing.
